### DISCUSSION TITLE
Error deleting account when two accounts have the same email.

### DISCUSSION THREAD

**Discussion by kjolley-zainar** [requarks/wiki#6769] [Error / Bug Report] [answered]
I created a "local" account that got an `id=16`. Later I enabled Google OAuth and it created an account with the same email but an `id=33`.  I wanted to do some testing and delete the `33` account. When I selected it for deletion it shows the `16` account. If I try to "Select User..." to get the `33` account it says I can't select the account I'm trying to delete. 

![Untitled](https://github.com/requarks/wiki/assets/117209556/4d71ae01-5920-4513-b7ba-8d3e1b562792)

(To add insult to injury, when I went to capture that error message I accidentally deleted the `16` account. The one I'm currently logged in as!  And trying to recreate it from the gui failed saying that email address is already taken.

**Comment by NGPixel** [comment, marked as answer]
In v2, you can have multiple accounts with the same email as long as they are different providers. This has been addressed in v3 (same account can have multiple providers).
